### ISSUE TITLE
Camel

### ISSUE THREAD

**Issue by tadayosi** [hawtio/hawtio-next#12]
Let's port https://github.com/hawtio/hawtio-integration/tree/main/plugins/camel

**Comment by tadayosi** [comment]
Hi @phantomjinx, thanks for your help!

Since this is a big task I split it into subtasks. Let's complete them one by one. As a starter, you can work on this sub task first:

- https://github.com/hawtio/hawtio-next/issues/17

I plan it as a feature for [0.2.0 release](https://github.com/hawtio/hawtio-next/milestone/3). We should first finish [0.1.0 release](https://github.com/hawtio/hawtio-next/milestone/1) with the basic core and JMX features, so that we can integrate Hawtio.next with the downstream [Hawtio](https://github.com/hawtio/hawtio) project as early as possible, but you can work on Camel plugin in parallel.

**Comment by tadayosi** [comment]
Done. Thanks everyone for your contributions!
